### PR TITLE
Simplification of shell-tests

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: Ilya Lobkov <ilya.lobkov@xored.com>

## Motivation and Context

Based on discussion in https://github.com/networkservicemesh/networkservicemesh/issues/2013
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by lobkovilya** [comment]
I deleted `scripts/helm-test-charts.sh` with some logic and I added `test/shell/*_test.sh` files that contain only real steps to reproduce some use-case. 
That's exactly what I described here https://github.com/networkservicemesh/networkservicemesh/issues/2013#issuecomment-567471469

File: .mk/k8s.mk [hunk 1]
```diff
@@ -143,10 +143,22 @@ k8s-forward:
 	@$(kubectl) port-forward $$($(kubectl) get pods | grep $(pod) | cut -d \  -f1) $(port1):$(port2)
 
 .PHONY: k8s-check
-k8s-check:
+k8s-check: k8s-icmp-check k8s-vpn-check
+
+.PHONY: k8s-icmp-check
```
Comment (denis-tingajkin) [inline .mk/k8s.mk]: Should we also update [example doc]( https://github.com/networkservicemesh/networkservicemesh/blob/7f7a7730a5d2ea0dce04c7819647458a92c35195/docs/vagrant/guide-vagrant.md#verify-deploying-the-examples) to use `k8s-icmp-check` / `k8s-vpn-check` ?

Comment (lobkovilya) [reply .mk/k8s.mk]: I didn't change the behaviour of `make k8s-check` it still runs 2 checks as it was before. Our bash-checkers also require some simplification, I intended to do it in further PRs

Comment (denis-tingajkin) [reply .mk/k8s.mk]: in the doc for verifying `icmp example`need to execute `make k8s-check`. Script `test/shell/icmp_test.sh` uses for verifying  `make k8s-icmp-check`. I think examples from docs and examples from CI should be equal.

**Pull request merged**